registration/models.py

```diff
@@ -82,6 +82,18 @@ def current_class(self):
             else "N/A"
         )
 
+    @property
+    def current_session(self):
+        return str(self.current_enrolment.session) if self.current_enrolment else "N/A"
+
+    @property
+    def current_preferred_class(self):
+        return (
+            self.current_enrolment.preferred_class.name
```

Comment:
when `preferred_class` is null, this'll throw an error: `AttributeError: 'NoneType' object has no attribute 'name'`

this'll be the case with `enrolled_class` above  as well – could we fix up both of those and maybe add tests for them in test_models.py?